analyzer/src/main/kotlin/managers/GoDep.kt

```diff
@@ -144,9 +145,9 @@ class GoDep(
 
         val scope = Scope("default", packageRefs.toSortedSet())
         val projectName = try {
-            val url = URL(projectVcs.url)
+            val uri = URI(projectVcs.url)
```

Comment:
The `MalformedURLException` below should then become `URISyntaxException`.

Would you mind replacing the other `URL` usage in this file with `URI`, too, so we can get rid of the `URL` import altogether?

Reply:
Done.